tests/app/main/test_metrics.py

```diff
@@ -0,0 +1,52 @@
+# coding=utf-8
+from tests.app.helpers import BaseApplicationTest
+import re
+
+
+def load_prometheus_metrics(response_bytes):
+    return dict(re.findall(b"(\w+{.+?}) (\d+)", response_bytes))
```

Comment:
:rotating_light: should be `br"...` here so the pattern's interpreted raw :rotating_light:

(or `rb"...` if you want to confuse everyone :gem: )

Reply:
(wonder if there's a flake8 check for this)

Reply:
What does the rawness do? Better for regexes?

Reply:
Happy to do by the wat 😂

Reply:
*way

Reply:
Although saying that I'm closing off this PR because I've done it fully in brief responses